Source/CarthageKit/Errors.swift

```diff
@@ -25,6 +25,7 @@ public enum CarthageErrorCode: Int {
 	case InvalidArchitectures
 	case MissingEnvironmentVariable
 	case NoSharedSchemes
+	case DuplicateDependenciesInPrivateCartfile
```

Comment:
How about just `DuplicateDependencies`? We could use this error to refer to any incidence of duplicate listings.